### ISSUE TITLE
Question about the strategy of negative category

### ISSUE THREAD

**Issue by YifanXu74** [IDEA-Research/GroundingDINO#74]
Hi, very impressive work! 
According to the paper, the loss function is similar to GLIP, and there is an [answer](https://github.com/IDEA-Research/GroundingDINO/issues/1#issuecomment-1480821693) about loss.  I notice that GLIP assigns the negative category (background) to the last token of the sentence (maybe [EOS] token). Am I right? Did GroundingDINO use the same strategy?

Thank you.

**Comment by SlongLiu** [comment]
We have negative samples during training. For example, we will sample some classes that do not exist in the image to construct a text prompt.

**Comment by jaideep11061982** [comment]
how to give negative prompt to exclude the object we dont want to get selected

**Comment by JosephChenHub** [comment]
@SlongLiu Hi Shilong, have you ever compare these two strategies for negative categories ? 
1.  prepare a dictionary that contains nouns occurred in the dataset, and random sample some words not occurred in the given image to used as negatives;
2. gather all nouns in a mini-batch, and negatives are those unmatched .
